<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConfigureDialog</class>
 <widget class="QDialog" name="DatabaseConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelDBType">
     <property name="text">
      <string>Database Type</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxDBType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxDBType">
     <item>
      <property name="text">
       <string>QSQLITE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QPSQL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QMYSQL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QOCI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QODBC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QDB2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QTDS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>QIBASE</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>DirectlyConectToLocalDBFile</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDBHost">
     <property name="text">
      <string>Host</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDBHost</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditDBHost">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 150);</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string>Port</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxPort</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelDBUser">
     <property name="text">
      <string>User</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDBUser</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditDBUser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 150);</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelDBPassword">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDBPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditDBPassword">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 150);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDBName">
     <property name="text">
      <string>DataBase Name</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDBName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditDBName">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButtonDBPath">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QSpinBox" name="spinBoxPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 150);</string>
     </property>
     <property name="maximum">
      <number>5600</number>
     </property>
     <property name="value">
      <number>1433</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatabaseConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
